projects/igniteui-angular/src/lib/slider/slider.component.ts

```diff
@@ -1004,6 +976,12 @@ export class IgxSliderComponent implements
         // Set track travel zone
         this._pMin = this.valueToFraction(this.lowerBound) || 0;
         this._pMax = this.valueToFraction(this.upperBound) || 1;
+
+        this._resizeNotify.pipe(takeUntil(this._destroyer$)).subscribe(() => {
+            this._ngZone.runTask(() => {
+                this.stepDistance = this.calculateStepDistance();
+            });
+        });
```

Comment:
Not a fan of triggering a zone run task for this callback.
IMO it would be better to leave the observer in the zone and throttle the resize observer events.